src/XeroPHP/Application/PartnerApplication.php

```diff
@@ -8,8 +8,8 @@ class PartnerApplication extends Application
 {
     protected static $_type_config_defaults = [
         'xero' => [
-            'site'     => 'https://api-partner.network.xero.com',
-            'base_url' => 'https://api-partner.network.xero.com',
+            'site'     => 'https://api.xero.com',
+            'base_url' => 'https://api.xero.com',
```

Comment:
I'm happy to merge most of it, just these two lines are redundant, since they're defined [here](https://github.com/calcinai/xero-php/blob/master/src/XeroPHP/Application.php#L17)